aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1730,6 +1781,34 @@ impl VMAdapter for AptosVM {
             },
         })
     }
+
+    fn execute_single_transaction_sequential(
+        &self,
+        txn: &PreprocessedTransaction,
+        view: &impl StateView,
+        log_context: &AdapterLogSchema,
+    ) -> Result<(VMStatus, VMOutput, Option<String>), VMStatus> {
+        match self.execute_single_transaction(txn, &view, log_context, true) {
+            Ok((vm_status, vm_output, sender)) => match vm_output.try_materialize(view) {
+                Ok(vm_output) => {
+                    return Ok((vm_status, vm_output, sender));
+                },
+                Err(vm_status) => {
+                    if matches!(vm_status, VMStatus::MoveAbort(_, code) if code == EADD_OVERFLOW || code == ESUB_UNDERFLOW)
+                    {
+                        return Err(vm_status);
+                    }
+                },
+            },
+            Err(vm_status) => {
+                if matches!(vm_status, VMStatus::MoveAbort(_, code) if code == EADD_OVERFLOW || code == ESUB_UNDERFLOW)
```

Comment:
This an above is wrong, it should be the negation of this

Reply:
Good catch. Fixed it.